service/gcs/bridge/connection.go

```diff
@@ -5,48 +5,18 @@ import (
 
 	"github.com/pkg/errors"
 
+	"io"
```

Comment:
Yep, that seems good to me! I didn't know there was a more standard convention, and I'm all for following that rather than the one we've been using in the GCS.